transformers/test_data/log_sorted_offer.go

```diff
@@ -0,0 +1,51 @@
+package test_data
+
+import (
+	"math/big"
+	"math/rand"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+	"github.com/makerdao/vulcanizedb/pkg/fakes"
+)
+
+var (
+	logSortedOfferOfferID, _ = new(big.Int).SetString("811648", 10)
```

Comment:
what's the motivation for putting this string into a big int? Seems like the only usage is calling `String()` on it?

Reply:
🤦‍♂️